README.md

```diff
@@ -913,6 +913,27 @@ $middlewares = [
 ];
 ```
 
+### JsonValidator
+
+Uses [justinrainbow/json-schema](https://github.com/justinrainbow/json-schema) to validate an `application/json` request body with a JSON schema:
+
+```php
+use Psr7Middlewares\Middleware;
+
+$middlewares = [
+
+    // Transform `application/json` into an object, which is a requirement of `justinrainbow/json-schema`.
+    Middleware::payload([
+        'forceArray' => false,
+    ]),
+    
+    // Specify a JSON file (publicly-accessible in this example), or a JSON string decoded into object-notation.
+    Middleware::jsonValidator((object) [
```

Comment:
oh, i could create a couple of named factories, one for a decoded object, and another that accepts SplFileObject (great suggestion).